apis/extension/scheduling.go

```diff
@@ -126,3 +129,34 @@ func RemoveReservationAllocated(pod *corev1.Pod, r *schedulingv1alpha1.Reservati
 	}
 	return false, nil
 }
+
+type DeviceAllocation struct {
+	Minor     int32
+	Resources corev1.ResourceList
+}
+
+type DevicesAllocation map[schedulingv1alpha1.DeviceType][]*DeviceAllocation
+
+func GetDevicesAllocation(pod *corev1.Pod) (*DevicesAllocation, error) {
+	deviceAllocations := &DevicesAllocation{}
```

Comment:
`DevicesAllocation` is map, so it should not be defined as pointer.

Reply:
fixed in #513